plenum/test/consensus/conftest.py

```diff
@@ -36,7 +34,7 @@ def _primary_in_view(view_no):
 @pytest.fixture
 def consensus_data(validators, primary, initial_view_no):
     def _data(name):
-        data = ConsensusSharedData(name, validators, 0)
+        data = ConsensusSharedData(name, 0)
```

Comment:
Again - don't forget to pass validators and primary to consensus shared data

Reply:
ok